enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BanDynamicVersions.java

```diff
@@ -99,9 +100,15 @@
     private boolean allowRanges;
 
     /**
-     * {@code true} if also optional dependencies should be checked
+     * {@code true} if ranges having the same upper and lower bound like {@code [1.0]} should be allowed.
+     * Only applicable if {@link #allowRanges} is not set to {@code true}.
      */
-    private boolean checkOptionals;
+    private boolean allowRangesWithSameUpperAndLowerBound;
```

Comment:
That is huge. What about `allowRangesWithIdenticalBounds?

Reply:
only slighly shorter, but I changed to that in https://github.com/apache/maven-enforcer/pull/187/commits/2d82f485ff5c73a6052b189fa9488d02fa4b3cbb